### PR TITLE
Configure include and exclude paths for ruff linter

### PR DESCRIPTION
**Pull request by jiridanek**

### CODE REVIEW

**Review by jstourac — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -62,7 +62,10 @@ ignore = [
 pythonVersion = "3.11"
 pythonPlatform = "Linux"
 
+# https://docs.astral.sh/ruff/configuration
 [tool.ruff]
+include = ["pyproject.toml", "ods_ci/**/*.py"]
+exclude = ["ods_ci/tests/Resources/Files/pipeline-samples/"]
```
[the remark this answers is not in the document]
Comment (jiridanek) [reply pyproject.toml:68]: I feel that it's best to copy them without making changes. One issue with them is the "imports at the beginning of the file". Currently, each step does its own importing, and I am unsure whether this actually is significant regarding functionality.

[the remark this answers is not in the document]
Comment (jiridanek) [reply pyproject.toml:68]: Also, they import things that we don't have defined in poetry project. Which is related to the mutual incompatibility of some of the packages involved.

[the remark this answers is not in the document]
Comment (jstourac) [reply pyproject.toml:68]: Right, point noted; would be nice if the project from which this is copied had some linting rules too, though... but whatever; thanks for this.

**Comment by jiridanek** [comment]
@jstourac looks like high-priority-review label does not recreate the required reviewers count, 
image

**Review by diegolovison — APPROVED** [review]

**Pull request merged**